### DISCUSSION TITLE
Disable Transifex workflow in a fork?

### DISCUSSION THREAD

**Discussion by mxmou** [ScratchAddons/ScratchAddons#924] [Help] [answered]
It's sending me notifications that it failed. Is it possible to disable it somehow?

**Comment by GrahamSH-LLK** [comment, marked as answer]
Go to the actions page, and disable it. 
![image](https://user-images.githubusercontent.com/64214252/101049416-7ff02600-3551-11eb-9faa-41a51ef12d01.png)

**Comment by Hans5958** [comment]
Sorry for the long overdue, but I did commit https://github.com/ScratchAddons/ScratchAddons/commit/5c851063751ca41209f44dca1d17e577941b1588 so you don't need to do this in the future,
